src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days >= 3 && days <= 6) {
+    total = days * 40 - 20;
```

Comment:
You can return without giving a value to a variable, like this
`return days * 40 - 20`